src/rrd.c

```diff
@@ -1220,7 +1289,7 @@ unsigned long long rrdset_done(RRDSET *st)
 						, st->id, rd->name
 						, st->current_entry
 						, t2
-						, get_storage_number_flags(rd->values[st->current_entry])
+						, /*get_storage_number_flags(rd->values[st->current_entry])*/ 0
```

Comment:
Well... what can I say... sorry again... it is a preparation to 
substitute that "strange" decimal floating point encoding to "double". I 
didn't realise this change was in the pull request...

On 28-04-2016 14:23, Costa Tsaousis wrote:
>
> In src/rrd.c 
> <https://github.com/firehol/netdata/pull/323#discussion_r61467462>:
>
> > @@ -1220,7 +1289,7 @@ unsigned long long rrdset_done(RRDSET *st)
> >  						, st->id, rd->name
> >  						, st->current_entry
> >  						, t2
> > -						, get_storage_number_flags(rd->values[st->current_entry])
> > +						, /*get_storage_number_flags(rd->values[st->current_entry])*/ 0
>
> this should show unpack the number and show the value - it is a debug()...
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/firehol/netdata/pull/323/files/460da7b0b8e59ebcb062b3b1627b452141c1a8ca..21b2e778bcce1d92816265f64175951556e53350#r61467462>
>